index.js

```diff
@@ -23,13 +23,16 @@ ${footer}`
 
   const processAsset = async (bundle, processFn) => {
     const {name} = bundle
-    const wrappingCode = await processFn({name, bundler})
 
-    if (wrappingCode) {
-      writeAsset(name, wrappingCode)
+    if (name !== undefined) {
+      const wrappingCode = await processFn({name, bundler})
+
+      if (wrappingCode) {
+        writeAsset(name, wrappingCode)
+      }
     }
 
-    bundle.childBundles.forEach(function(bundle) {
+    bundle.childBundles.forEach((bundle) => {
       processAsset(bundle, processFn)
```

Comment:
@asos-albinotonnina I think @dbilgili is saying that when `processAsset` is called recursively here, `bundle.name` is defined. So the child bundles are traversed in this `forEach` call, right?

Reply:
Exactly.

Reply:
fine for me then